r/neotoma_tutorial.Rmd

```diff
@@ -382,6 +474,7 @@ extinct_mammals$taxon.name <- trimws(extinct_mammals$taxon.name)
 ## and also requires manual interaction
 ## you can download data set below
 ## Manual selection key: 1, 0, 2, 0, 2, 1 
+## [TODO] I needed one more number for the manual selection. I put NA and then get 26 not classified below.
```

Comment:
Can you double check this?

Reply:
I'm not sure we should include this whole part in the tutorial. It is a little crazy.  It's not our fault, just the API and taxize together is pretty clunky....I do want to include somehow though.  I might just have two notebooks and we will go through the shorter one with the data read in option for the workshop.  Then I will have the longer one if they are interested.

Reply:
I like that idea. It'll save us time, but then they'll have access to the full pipeline.

Reply:
It' a plan!